pennylane/ops/op_math/controlled_ops.py

```diff
@@ -117,21 +132,35 @@ class ControlledQubitUnitary(ControlledOp):
 
     @classmethod
     def _unflatten(cls, data, metadata):
-        return cls(
-            data[0], control_wires=metadata[0], control_values=metadata[1], work_wires=metadata[2]
-        )
+        return cls(data[0], metadata[0], control_values=metadata[1], work_wires=metadata[2])
 
     # pylint: disable=arguments-differ, too-many-arguments, unused-argument, too-many-positional-arguments
     @classmethod
     def _primitive_bind_call(
         cls,
         base,
-        control_wires: WiresLike,
-        wires: WiresLike = (),
+        wires: WiresLike = None,
+        control_wires: WiresLike = "unset",
```

Comment:
```suggestion
        control_wires: WiresLike = "unset",
        wires: WiresLike = None,
```
Let's still keep the original order or there will be pure positional legacy usages that fail completley